pyshop/lp_model/lp_model.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+
+from .row import *
```

Comment:
Bytt ut import* med 
_from .row import RowBuilder, RowTypeBuilder_ 
osv., så blir det lettere å se hvor ting kommer fra

Reply:
Fixed in latest commit